Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:14-buster-slim
```

Comment:
Let's move the dockerfile into the devcontainer directory. We try to keep the top level small.